src/components/SearchForm/SearchForm.tsx

```diff
@@ -1,7 +1,7 @@
 import { SearchBar } from "@nypl/design-system-react-components"
 import { useRouter } from "next/router"
-import type { SyntheticEvent, Dispatch, SetStateAction } from "react"
-import { useContext, useState, useEffect } from "react"
+import type { SyntheticEvent, Dispatch, SetStateAction, RefObject } from "react"
```

Comment:
Don't think you need `RefObject`, right?

Reply:
removed